config/routes.rb

```diff
@@ -9,8 +9,22 @@
   get 'rentals/:id/invoice', to: 'rentals#invoice', as: 'rental_invoice'
   get 'rentals/safety_pdf'
   get 'rentals/training_pdf'
-  resources :rentals
```

Comment:
Aren't there resourceful routes for the rentals?

Reply:
This line is still there, just below the help routes. I guess it got inserted weird, I'll move it back.

Reply:
Ah alright I just missed that fact.